logstash-filter-verifier.go

```diff
@@ -50,6 +50,10 @@ var (
 			Flag("logstash-output", "Print the debug output of logstash.").
 			Default("false").
 			Bool()
+	logstashArgs = kingpin.
+			Flag("logstash-arg", "Command line arguments, which are passed to Logstash. Flag and value have to be provided as a flag each, e.g.: --logstash-arg=-n --logstash-arg=InstanceName").
+			Short('l').
```

Comment:
Done! My thinking was, that due to the repetitive form of this flag, it would be nice to have the short form. But I am also fine with the long form.